src/components/PatchesPage/PatchCard.tsx

```diff
@@ -167,8 +171,12 @@ const Left = styled(Center)`
 `;
 
 const Right = styled.div`
+  width: 110px;
   display: flex;
   justify-content: flex-end;
+  > button {
+    margin-left: ${size.xs};
+  }
```

Comment:
Since this is already using flexbox, you can use the `gap` property instead
```suggestion
  gap: ${size.xs};
```